gammapy/data/observations.py

```diff
@@ -259,6 +259,14 @@ def observation_dead_time_fraction(self):
         """
         return 1 - self.obs_info["DEADC"]
 
+    @lazyproperty
+    def obs_info(self):
+        """Observation info dictionary."""
```

Comment:
This makes the lines further down obsolete and takes a different approach: only using the events header if it is present completely ignoring any `obs_info`. 

The code below *merges* obs_info and events table, giving preference to the events header.

We should decide what to do here and then use `self.obs_info` in `fixed_pointing_info`.